scripts/setvolume.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+## Sets the volume for source $1 to $2 percent
+
+# $1 is the sound source name from "pacmd list-sources | grep 'name:'" without the angle brackets <>
+
+max=$(pacmd list-sources | perl -n0777E "say \$1 if /^.*name: <$1>.*?base volume: ([0-9]+).*/s")
```

Comment:
Dependency on perl not verified in CMakeLists.

Reply:
Good point.
Should it really be checked in CMakeLists? Or it should only be added to dependendies in the README?

Reply:
Should be added in both.

Reply:
Should be added in both